modules/lvms-limitations-to-configure-size-of-devices.adoc

```diff
@@ -23,36 +23,57 @@ ifdef::microshift[]
 endif::microshift[]
 
 ifndef::microshift[]
-.Size limits for different architectures using the default PE and LE size
-[cols="1,1,1,1,1", width="100%", options="header"]
+
+The following tables describe the chunk size and volume size limits for static and host configurations:
+
+.Tested configuration
+[cols="1,1", width="100%", options="header"]
+|====
+
+|Parameter
+|Value
+
+|Chunk size
+|128 KiB
+
+|Maximum volume size
+|32 TiB
+
+|====
+
+.Theoretical size limits for static configuration
+[cols="1,1,1", width="100%", options="header"]
+|====
+
+|Parameter
+|Minimum value
+|Maximum value
+
+|Chunk size
+|64 KiB
+|1 GiB
+
+|Volume size
+|Minimum size of the underlying {op-system-first} system.
+|Maximum size of the underlying {op-system} system.
+
 |====
-|Architecture
-|RHEL 6
-|RHEL 7
-|RHEL 8
-|RHEL 9
 
-|32-bit
-|16 TB
-|-
-|-
-|-
+.Theoretical size limits for a host configuration
+[cols="1,1", width="100%", options="header"]
+|====
 
-|64-bit
+|Parameter
+|Value
 
-|8 EB ^[1]^
+|Chunk size
+|This value is based on the configuration in the `lvm.conf` file. By default, this value is set to 128 KiB.
```

Comment:
```suggestion
|This value is based on the configuration in the `lvm.conf` file. By default, this value is set to `128 KiB`.
```

Not sure what the exact syntax would be in the config file.

Reply:
@jakobmoellerdev 
Can the users manually cofigure `lvm.conf` file? We haven't talked about "configuring the lvm.conf file" in our existing doc. Can this cause any confusion or lead to users raising doc bugs?

Reply:
As confirmed by the SME @jakobmoellerdev in slack:
 "This modification is described in the lvm documentation of Red Hat and should be out of scope for LVMS FWIW"